cmd/cluster/dynatrace/hcpGatherLogsCmd.go

```diff
@@ -146,7 +146,7 @@ func dumpEvents(deploys *appsv1.DeploymentList, parentDir string, targetNS strin
 		err = getEvents(DTURL, accessToken, eventsRequestToken, f)
 		f.Close()
 		if err != nil {
-			return fmt.Errorf("failed to get logs %v", err)
+			fmt.Println(fmt.Errorf("failed to get logs %v. Query: %v", err, eventQuery.finalQuery))
```

Comment:
```suggestion
			log.Printf("failed to get logs, continuing: %v. Query: %v", err, eventQuery.finalQuery)
```
I'm also ok to use `fmt.Printf`, but prefer `log.Printf` to print to `stderr`. The main thing is taking out the `fmt.Errorf` and mentioning that we are "continuing"/skipping/ignoring an error explicitly